mailpoet/tests/_support/AcceptanceTester.php

```diff
@@ -577,8 +580,13 @@ public function selectPaymentMethod($method = 'cod') {
    */
   public function placeOrder() {
     $i = $this;
-    $i->waitForText('Place order');
-    $i->click('Place order');
+    // Add a note to order just to avoid flakiness due to race conditions
+    $i->click(Locator::contains('label', 'Add a note to your order'));
+    $i->fillField('.wc-block-components-textarea', 'This is a note');
+    $i->waitForElementClickable(Locator::contains('button', 'Place Order'));
+    $i->waitForText('Place Order');
+    $i->waitForElementClickable(Locator::contains('button', 'Place Order'));
```

Comment:
Is it necesssary to wait for the same element to be clickable twice and also check for the presence of the text `Place order`? I'm wondering if the three lines above can be simplified into just one.

Reply:
No, this is definitely a leftover. Good catch, will be fixed.